packages/eslint-plugin/docs/rules/explicit-module-boundary-types.md

```diff
@@ -12,9 +12,11 @@ It can also improve TypeScript type checking performance on larger codebases.
 
 ## Examples
 
+### 1. Exported members must have explicit signatures
+
 <!--tabs-->
```

Comment:
Looking at this first pair of tabs:

1. ✏️ `export function test() {`: Great!
2. ✏️ `export default function () {`: Odd that the _Correct_ version is different. I'd say it should probably instead have an explicit return type.
3. `export var arrowFn`: Great!
   - ✏️ I wonder if we can delete number 2 (`export default function() {`), since this `arrowFn` also serves the purpose of showing different syntax?
4. `export var arrowFn = (arg):`: Great!
5. `export var arrowFn = (arg: any):`: Great!
6. ✏️ `export class Test {`: Mostly great, but can you move the _Correct_ code's comment to just above `method()`? It's confusing IMO to see it change location (and the new location is inaccurate to what the comment says)

✏️ indicates a request for changes :)